wrappers/cabal

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -o xtrace
+
```

Comment:
Is it safe to keep this in? Or will we add more undesired logs to HLS?

Reply:
it's safe in the sense that it's specified by posix: https://pubs.opengroup.org/onlinepubs/007904875/utilities/set.html

Whether it's desired, I don't know.

Reply:
I just checked... it spams the output really hard: https://paste.tomsmeding.com/WllOnqGK

Reply:
Ah thank you! Going to remove it then.